specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2018-06-01/mariadb.json

```diff
@@ -1786,6 +1794,10 @@
               "$ref": "#/definitions/SslEnforcement",
               "description": "Enable ssl enforcement or not when connect to server."
             },
+            "publicNetworkAccess": {
```

Comment:
This is typically regarded as a breaking change to a GA API. Why not publish a new API version?